.gitignore

```diff
@@ -12,13 +12,16 @@ test/basic/*.out
 test/subset-paths/*.out
 
 test/temp.*
+test/*.n
 test/*.chop
 test/*.crush
+test/*/*.crush
```

Comment:
FWIW, you can gitignore stuff in *any* directory with `**`. Like `test/**/*.crush` to subsume these two lines. Or you can just put a separate `.gitignore` in the `test` directory to avoid repeating `test/` on every line and just put `*.crush` in there to cover all subdirectories.

For details, see: https://git-scm.com/docs/gitignore#_pattern_format

Reply:
Many thanks!